libs/ml/tests/unit/ops/max_pool.cpp

```diff
@@ -19,6 +19,7 @@
 #include "core/serializers/main_serializer_definition.hpp"
 #include "math/base_types.hpp"
 #include "ml/ops/max_pool.hpp"
+#include "ml/ops/max_pool_2d.hpp"
```

Comment:
I don't want to move back ml/ops/max_pool_2d.hpp to ml/ops/max_pool.hpp but if it is needed please let me know